hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -241,6 +241,18 @@
       sw_images: ["sw/device/tests/rv_timer_smoketest:1"]
       en_run_modes: ["sw_test_mode"]
     }
+    {
```

Comment:
@sriyerg as requested, please confirm for correctness

Reply:
Looks right, thanks.